packages/core-react/src/components/TextField/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+
+export default function TextField({
+  id,
+  type,
+  placeholder,
+  disabled,
+  required,
+  value,
+  error,
+  compact,
+  label,
+  ...rest
+}) {
+  const wrapperClass = clsx('ray-text-field', {
```

Comment:
Should this also handle textarea?

    'ray-text-field': !textarea,
    'ray-text-area': textarea,

Reply:
maaybeee.